extensions/opc/src/fetchopc.cpp

```diff
@@ -194,11 +204,25 @@ namespace processors {
     {
       try {
         opc::NodeData nodedata = connection_->getNodeData(ref);
-        OPCData2FlowFile(nodedata, context, session);
-        variablesFound_++;
+        bool write = true;
+        if (lazy_mode_) {
+          write = false;
+          std::string nodeid = nodedata.attributes["Full path"];
+          std::string cur_timestamp = node_timestamp_[nodeid];
+          std::string new_timestamp = nodedata.attributes["Sourcetimestamp"];
+          if (cur_timestamp != new_timestamp) {
+            node_timestamp_[nodeid] = new_timestamp;
+            logger_->log_warn("Node %s has new source timestamp %s", nodeid, new_timestamp);
```

Comment:
I think a warning level log is excessive here, as this is a normal code path. I would leave it at max on info, but if it spams too much even debug.

Reply:
Agreed, I vote for debug here.